f2/src/theme/canada.js

```diff
@@ -25,6 +25,7 @@ const colors = {
     700: '#373737',
     800: '#202020',
     900: '#191919',
+    skipLink: '#444444',
```

Comment:
Do we need a specific color key just for skiplinks?

Reply:
not if we're using `blue.900` 😄